src/main/scala/chisel3/UnitTest.scala

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import chisel3.experimental.BaseModule
+import chisel3.experimental.hierarchy.Definition
+import java.io.File
+import java.util.jar.JarFile
+import scala.collection.JavaConverters._
+
+/** All classes and objects marked as `UnitTest` are automatically discoverable
+ *  by the `DiscoverUnitTests` helper. */
```

Comment:
Nit: Usually I've seen multi-line comments look like:

```suggestion
/** All classes and objects marked as `UnitTest` are automatically discoverable
  *  by the `DiscoverUnitTests` helper. 
  */
```

If you run reformat, this may be fixed: `./mill __.reformat`.

Reply:
It's a shame that `./mill __.reformat` doesn't reformat comments properly (at least on my machine) :cry: